lib/logstash/inputs/rabbitmq.rb

```diff
@@ -121,8 +121,12 @@ class RabbitMQ < LogStash::Inputs::Threadable
       # restart).
       config :exclusive, :validate => :boolean, :default => false
 
-      # Extra queue arguments as an array.
-      # To make a RabbitMQ queue mirrored, use: `{"x-ha-policy" => "all"}`
+      # Optional queue arguments as an array.
```

Comment:
can you add this new comment to [docs/index.asciidoc](https://github.com/logstash-plugins/logstash-input-rabbitmq/blob/master/docs/index.asciidoc#plugins-inputs-rabbitmq-arguments)?
You can even delete this whole comment here in the ruby file, as the asciidoc is now the source of truth for plugin docs.